kolibri/core/discovery/api.py

```diff
@@ -34,6 +37,27 @@ class NetworkLocationViewSet(viewsets.ModelViewSet):
         "instance_id",
     ]
 
+    def get_queryset(self):
+        syncable = self.request.query_params.get("syncable", None)
+        if syncable == "1":
+            # Include KDP's reserved location
+            queryset = NetworkLocation.objects.filter(
+                Q(location_type=LocationTypes.Static)
+                | Q(id=DATA_PORTAL_BASE_INSTANCE_ID)
+            )
+        elif syncable == "0":
+            # Include Studio's reserved location
+            queryset = NetworkLocation.objects.filter(
+                Q(location_type=LocationTypes.Static)
+                | Q(id=CENTRAL_CONTENT_BASE_INSTANCE_ID)
+            )
+        else:
+            # Exclude both KDP and Studio
+            queryset = NetworkLocation.objects.filter(
+                location_type=LocationTypes.Static
+            )
+        return queryset
```

Comment:
What about dynamic locations? These filters don't seem to able to return those

Reply:
I think I incorrectly assumed that only static network locations should be returned, looking back at this, I'm not completely sure why. I will update the filter to return dynamic locations as well.

Reply:
As a reminder:
- Static: locations manually added by users
- Dynamic: locations detected on a local network
- Reserved: well-known locations in our ecosystem (essentially to isolate them from the other two types)